src/devtools/useAtomsSnapshot.ts

```diff
@@ -3,12 +3,15 @@ import { SECRET_INTERNAL_getScopeContext as getScopeContext } from 'jotai'
 import type { Atom, Scope } from '../core/atom'
 import {
   DEV_GET_ATOM_STATE,
+  DEV_GET_MOUNTED,
   DEV_GET_MOUNTED_ATOMS,
   DEV_SUBSCRIBE_STATE,
 } from '../core/store'
-import type { AtomState, Store } from '../core/store'
+import type { AtomState, Mounted, Store } from '../core/store'
 
-type AtomsSnapshot = Map<Atom<unknown>, unknown>
+type AtomsValues = Map<Atom<unknown>, unknown>
+type AtomsDependants = Map<Atom<unknown>, Set<Atom<unknown>>>
+type AtomsSnapshot = readonly [AtomsValues, AtomsDependants]
```

Comment:
So, we are changing our terminology. We will need to follow it in `useGotoAtomsSnapshot`.